src/GistEntropyUtils.h

```diff
@@ -0,0 +1,88 @@
+#ifndef GIST_ENTROPY_UTILS_H
+#define GIST_ENTROPY_UTILS_H
+
```

Comment:
Ideally we could put all of this in a namespace, e.g.
```C++
namespace Cpptraj {
namespace GistEntropyUtils {
```
I've done this with the clustering routines recently (see the Cluster subdirectory) and I think it makes for nicer code. For some things however it doesn't work since I made `Cpptraj` a class (when I started cpptraj I knew far less about good C++ practices than I do now). I think since only the GIST action will pull this in it would be ok. Let me know what you think.